### ISSUE TITLE
[Feature request] Implement Android Pluggable Transport

### ISSUE THREAD

**Issue by xtianxian** [schwabe/ics-openvpn#1339]
Please implement the Android Pluggable Transport in your app so we can use obfs4 and other transports. 
Thank you!

**Comment by schwabe** [comment]
This will have a low priority from my side. OpenVPN itself has no support for pluggable transports at the moment, so expect this only after OpenVPN/OpenVPN3 have support for that.
